### PR TITLE
Added 2 programs given on 27-11-21

### PR DESCRIPTION
**Pull request by harshitjain01**
This is a test commit. Please check if it's the appropriate way of doing it.

### CODE REVIEW

**Comment by BudhirajaMadhav** [comment]
Dont use n, use meaningful variables

File: Harshit_Jain/fibonacci.cpp [hunk 1]
```diff
@@ -0,0 +1,31 @@
+#include <iostream>
+using namespace std;
+
```
Comment (BudhirajaMadhav) [inline Harshit_Jain/fibonacci.cpp:3]: Dont use n

**Review by BudhirajaMadhav — CHANGES_REQUESTED** [review]

File: Harshit_Jain/fibonacci.cpp (hunk 1 above)
Comment (harshitjain01) [reply Harshit_Jain/fibonacci.cpp:3]: Updated. Please check.

File: Harshit_Jain/gcd_lcm.cpp [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+        n1=n1-n2;
+        
+        else
+        n2=n2-n1;
+    }
+    
+    cout<<n1<<endl;             //GCD
+    
+    cout<<(num1*num2)/n1;       //gcd*lcm=num1*num2
+}
```
Comment (BudhirajaMadhav) [inline Harshit_Jain/gcd_lcm.cpp]: Don't use n1, n2. If a comment is done somewhere then fix that issue in every file.

**Review by BudhirajaMadhav — CHANGES_REQUESTED** [review]

**Review by BudhirajaMadhav — APPROVED** [review]

File: Harshit_Jain/gcd_lcm.cpp (hunk 2 above)
Comment (harshitjain01) [reply Harshit_Jain/gcd_lcm.cpp]: Made the required changes.

**Pull request merged**